tests/python/tensorrt/test_tensorrt_lenet5.py

```diff
@@ -46,7 +46,7 @@ def run_inference(sym, arg_params, aux_params, mnist, all_test_labels, batch_siz
     # Get this value from all_test_labels
     # Also get classes from the dataset
     num_ex = 10000
-    all_preds = np.zeros([num_ex, 10])
+    all_preds = np.zeros([num_ex, 10], dtype=np.float32)
```

Comment:
isnt this the default? Why do we need to explicitly specify fp32?

Reply:
yes by default it is np.float64.

Reply:
Why do we need to specify the type here? Arent we just going to override later with the outputs:

```
all_preds[offset:offset+extent, :] = executor.outputs[0].asnumpy()[:extent]
```

Reply:
No it wont come from outputs
```
>>> x = np.zeros((2, 3))
>>> y = np.zeros((2, 3), dtype=np.float32)
>>> x[:] = y
>>> x
array([[ 0.,  0.,  0.],
       [ 0.,  0.,  0.]])
>>> x.dtype
dtype('float64')
```